crates/matrix-sdk-ui/src/room_list_service/filters/all.rs

```diff
@@ -0,0 +1,26 @@
+use matrix_sdk::RoomListEntry;
+
+/// Create a new filter that will accept all filled or invalidated entries.
+pub fn new_filter() -> impl Fn(&RoomListEntry) -> bool {
+    |room_list_entry| -> bool {
+        matches!(room_list_entry, RoomListEntry::Filled(_) | RoomListEntry::Invalidated(_))
```

Comment:
I find it weird that this filter is called "all" but still excludes some entries.

Reply:
Yeah, that’s a technical details. `Empty` are always filtered out because they are never rendered.